### PR TITLE
Improved examples

### PR DESCRIPTION
**Pull request by sapegin**
1. Just tiny tweaks.
2. Completely new iterative button example.

### CODE REVIEW

File: src/components/core/Button/Button.js [hunk 1]
```diff
@@ -40,7 +42,9 @@ const Button = styled(Box).attrs({
 `;
 
 Button.propTypes = {
+	/** Button text */
```
Comment (okonet) [inline src/components/core/Button/Button.js]: Text -> Label

File: src/components/core/Button/Button.js [hunk 2]
```diff
@@ -40,7 +42,9 @@ const Button = styled(Box).attrs({
...
 	children: PropTypes.node,
+	secondary: PropTypes.bool,
```
Comment (okonet) [inline src/components/core/Button/Button.js]: I’d say it should be secondary be default and `primary` as a prop

File: src/exercises/exercise02/Readme.md [hunk 3]
```diff
@@ -1,20 +1,253 @@
...
 
 ### Getting started
 
 Run Styleguidist: `npm run exercise 2`.
 
-### Task
+### Tasks
 
-1.  Change base tag from `button` to `a`, add `href` props to examples in `Button.md`.
+#### 2.1. Introducing Styled Components
```
Comment (okonet) [inline src/exercises/exercise02/Readme.md]: “styled-components”

**Review by okonet — APPROVED** [review]

**Comment by okonet** [comment]
I’ve added some comments

File: src/components/core/Button/Button.js (hunk 1 above)
Comment (sapegin) [reply src/components/core/Button/Button.js]: 👍

File: src/exercises/exercise02/Readme.md (hunk 3 above)
Comment (sapegin) [reply src/exercises/exercise02/Readme.md]: 😢

File: src/components/core/Button/Button.js (hunk 2 above)
Comment (sapegin) [reply src/components/core/Button/Button.js]: Good point!

**Comment by sapegin** [comment]
I've also added a result at the beginning:

image 2018-05-28 at 10 04 16 pm

**Review by okonet — APPROVED** [review]

**Comment by okonet** [comment]
LGTM!

**Pull request merged**